examples/decode_oneshot.cc

```diff
@@ -16,8 +16,9 @@
 
 #include "jxl/decode.h"
 #include "jxl/decode_cxx.h"
+#include "jxl/resizable_parallel_runner.h"
+#include "jxl/resizable_parallel_runner_cxx.h"
 #include "jxl/thread_parallel_runner.h"
```

Comment:
do you still need this include? I have the impression that we should use either one or the other.